pygmt/src/inset.py

```diff
@@ -46,45 +52,12 @@ def inset(
 
     Parameters
     ----------
-    position : str or list
-        *xmin/xmax/ymin/ymax*\ [**+r**][**+u**\ *unit*]] \
-        | [**g**\|\ **j**\|\ **J**\|\ **n**\|\ **x**]\ *refpoint*\
-        **+w**\ *width*\ [/*height*][**+j**\ *justify*]\
-        [**+o**\ *dx*\ [/*dy*]].
-
-        *This is the only required parameter.*
-        Define the map inset rectangle on the map. Specify the rectangle
-        in one of three ways:
-
-        Append **g**\ *lon*/*lat* for map (user) coordinates,
-        **j**\ *code* or **J**\ *code* for setting the *refpoint* via a
-        :doc:`2-character justification code </techref/justification_codes>`
-        that refers to the (invisible)
-        projected map bounding box, **n**\ *xn*/*yn* for normalized (0-1)
-        bounding box coordinates, or **x**\ *x*/*y* for plot
-        coordinates (inches, centimeters, points, append unit).
-        All but **x** requires both ``region`` and ``projection`` to be
-        specified. You can offset the reference point via
-        **+o**\ *dx*/*dy* in the direction implied by *code* or
-        **+j**\ *justify*.
-
-        Alternatively, give *west/east/south/north* of geographic
-        rectangle bounded by parallels and meridians; append **+r** if the
-        coordinates instead are the lower left and upper right corners of
-        the desired rectangle. (Or, give *xmin/xmax/ymin/ymax* of bounding
-        rectangle in projected coordinates and optionally
-        append **+u**\ *unit* [Default coordinate unit is meters (**e**)].
```

Comment:
`inset`'s `-D` option can also accept *west/east/south/north*. An example is:
```
gmt begin map png
    gmt coast -Rg -JH15c -Baf -Ggray
    gmt inset begin -D100/130/30/60 -F
    gmt inset end
gmt end show
```
The output is:
<img width="1836" height="892" alt="Image" src="https://github.com/user-attachments/assets/7fde9c2b-29c1-44f9-9919-6ca528b49169" />

The syntax is a little weird, so I've remove it from the docstrings.